root/static/scripts/common/MB/Control/Autocomplete.js

```diff
@@ -588,8 +589,6 @@ $.widget("ui.menu", $.ui.menu, {
              */
             event.stopPropagation();
             event.preventDefault();
-
-            return true;
```

Comment:
There's a case below where we check `if (!this._selectAction(event)) {`, so this seems off and we should rather return false at the end.